fastlane/Fastfile

```diff
@@ -135,9 +142,6 @@ before_all do |lane|
   # Skip these checks/steps for test lane (not needed for testing)
   next if lane == :test_without_building
 
-  # Ensure we use the latest version of the toolkit
-  check_for_toolkit_updates unless is_ci || ENV['FASTLANE_SKIP_TOOLKIT_UPDATE_CHECK']
-
```

Comment:
I moved this check to the top of the `code_freeze` lane instead of `before_all`. It hasn't been useful to have it run before every single lane during the course of a release.